.travis.yml

```diff
@@ -1,88 +1,35 @@
 language: python
 python:
-  - "2.6"
   - "2.7"
-  - "3.2"
-  - "3.3"
-  - "3.4"
   - "3.5"
   - "3.6"
+  - "3.7"
+  - "3.8"
 env:
-  - DJANGO_VERSION=">=1.4.0,<1.5.0"
-  - DJANGO_VERSION=">=1.5.0,<1.6.0"
-  - DJANGO_VERSION=">=1.6.0,<1.7.0"
-  - DJANGO_VERSION=">=1.7.0,<1.8.0"
-  - DJANGO_VERSION=">=1.8.0,<1.9.0"
-  - DJANGO_VERSION=">=1.9.0,<1.10.0"
-  - DJANGO_VERSION=">=1.10.0,<1.11.0"
   - DJANGO_VERSION=">=1.11.0,<2.0.0"
   - DJANGO_VERSION=">=2.0.0,<2.1.0"
   - DJANGO_VERSION=">=2.1.0,<2.2.0"
+  - DJANGO_VERSION=">=2.2.0,<2.3.0"
+  - DJANGO_VERSION=">=3.0.0,<3.1.0"
 
 matrix:
   exclude:
-    - python: "3.2"
-      env: DJANGO_VERSION=">=1.4.0,<1.5.0"
-    - python: "3.2"
-      env: DJANGO_VERSION=">=1.9.0,<1.10.0"
-    - python: "3.2"
-      env: DJANGO_VERSION=">=1.10.0,<1.11.0"
-    - python: "3.2"
-      env: DJANGO_VERSION=">=1.11.0,<2.0.0"
-    - python: "3.2"
-      env: DJANGO_VERSION=">=2.0.0,<2.1.0"
-    - python: "3.2"
-      env: DJANGO_VERSION=">=2.1.0,<2.2.0"
-    - python: "3.3"
-      env: DJANGO_VERSION=">=1.4.0,<1.5.0"
-    - python: "3.3"
-      env: DJANGO_VERSION=">=1.9.0,<1.10.0"
-    - python: "3.3"
-      env: DJANGO_VERSION=">=1.10.0,<1.11.0"
-    - python: "3.3"
-      env: DJANGO_VERSION=">=1.11.0,<2.0.0"
-    - python: "3.3"
-      env: DJANGO_VERSION=">=2.0.0,<2.1.0"
-    - python: "3.3"
-      env: DJANGO_VERSION=">=2.1.0,<2.2.0"
-    - python: "3.4"
-      env: DJANGO_VERSION=">=1.4.0,<1.5.0"
-    - python: "3.4"
-      env: DJANGO_VERSION=">=2.1.0,<2.2.0"
-    - python: "3.5"
-      env: DJANGO_VERSION=">=1.4.0,<1.5.0"
-    - python: "3.5"
-      env: DJANGO_VERSION=">=1.5.0,<1.6.0"
     - python: "3.5"
-      env: DJANGO_VERSION=">=1.6.0,<1.7.0"
-    - python: "3.5"
-      env: DJANGO_VERSION=">=1.7.0,<1.8.0"
-    - python: "3.6"
-      env: DJANGO_VERSION=">=1.4.0,<1.5.0"
-    - python: "3.6"
-      env: DJANGO_VERSION=">=1.5.0,<1.6.0"
-    - python: "3.6"
-      env: DJANGO_VERSION=">=1.6.0,<1.7.0"
-    - python: "3.6"
-      env: DJANGO_VERSION=">=1.7.0,<1.8.0"
-    - python: "2.6"
-      env: DJANGO_VERSION=">=1.7.0,<1.8.0"
-    - python: "2.6"
-      env: DJANGO_VERSION=">=1.8.0,<1.9.0"
-    - python: "2.6"
-      env: DJANGO_VERSION=">=1.9.0,<1.10.0"
-    - python: "2.6"
-      env: DJANGO_VERSION=">=1.10.0,<1.11.0"
-    - python: "2.6"
+      env: DJANGO_VERSION=">=3.0.0,<3.1.0"
+    - python: "3.8"
       env: DJANGO_VERSION=">=1.11.0,<2.0.0"
-    - python: "2.6"
+    - python: "3.8"
       env: DJANGO_VERSION=">=2.0.0,<2.1.0"
-    - python: "2.6"
+    - python: "3.8"
       env: DJANGO_VERSION=">=2.1.0,<2.2.0"
     - python: "2.7"
       env: DJANGO_VERSION=">=2.0.0,<2.1.0"
     - python: "2.7"
       env: DJANGO_VERSION=">=2.1.0,<2.2.0"
```

Comment:
This is up to the maintainer, but 2.2 is the last LTS release and they advise dropping support for versions lower than 2.2. LTS for 2.2 April 2022.

> 
Following the release of Django 3.0, we suggest that third-party app authors drop support for all versions of Django prior to 2.2. 

https://docs.djangoproject.com/en/3.0/releases/3.0/

Reply:
I added my thoughts below - https://github.com/waveaccounting/dj-inmemorystorage/pull/17/files#r360966204

This package is relatively small and doesn't change too much so I'm OK supporting a bit more/longer than strictly suggested until it becomes a burden/compatibility nightmare.